README.md

```diff
@@ -204,7 +204,7 @@ Resource name is written in the plural form.
 
 |Parameter|Description|Default|Example|
 |---------|-----------|-------|-------|
-|`ids=`|Item IDs|(empty)|`1,2,5`|
+|`q[field_name]=`|A unique query parameter for each field for filtering|(empty)|`1,2,5`|
```

Comment:
Maybe it's more useful to suggest example with query key,

`q[id]=1,2,5`, `q[admin]=true&q[registered]=true`